### PR TITLE
Use blank type for JSONAny in OpenAPI

### PR DESCRIPTION
**Pull request by awrichar**
Type is unknown, as it may be any valid JSON.

See https://github.com/hyperledger/firefly-sdk-nodejs/issues/18

### CODE REVIEW

File: internal/oapispec/openapi3.go [hunk 1]
```diff
@@ -155,20 +155,20 @@ func ffTagHandler(ctx context.Context, route *Route, name string, tag reflect.St
...
+	case "FFTime":
+		schema.Type = typeString
+		schema.Format = "date-time"
+	case "Bytes32":
+		schema.Type = typeString
+		schema.Format = "byte"
+	case "FFBigInt":
+		schema.Type = typeString
+	case "JSONAny":
+		schema.Type = ""
```
Comment (peterbroadhurst) [inline internal/oapispec/openapi3.go:171]: Should this not be: https://swagger.io/docs/specification/data-models/data-types/#any

**Review by peterbroadhurst — CHANGES_REQUESTED** [review]
If possible to use the any type

File: internal/oapispec/openapi3.go (hunk 1 above)
Comment (awrichar) [reply internal/oapispec/openapi3.go:171]: Seems appropriate - I'll look into it

Comment (awrichar) [reply internal/oapispec/openapi3.go:171]: I think this is actually what I'm doing. That link says to omit the type if you want to allow "any type". As a reference, for something like a contract `location`, this is what is written (notice the omission of "type" similar to the examples in the docs):

```
location:
  description: If this API is tied to an individual instance of
    a smart contract, this field can include a blockchain specific
    contract identifier. For example an Ethereum contract address,
    or a Fabric chaincode name and channel
```

Comment (peterbroadhurst) [reply internal/oapispec/openapi3.go:171]: Thanks @awrichar - makes sense 👍

**Review by peterbroadhurst — APPROVED** [review]

**Pull request merged**